### ISSUE TITLE
LegendasDivx.pt Uncaught error

### ISSUE THREAD

**Issue by jaum20** [morpheus65535/bazarr#2009]
**Describe the bug**

**To Reproduce**
1. add LegendasDivx.pt as a provider
2. add login, password
3. search for some subs

**Expected behavior**
login to the server and download subtitles
**Screenshots**

**Software (please complete the following information):**
 - Bazarr: v1.1.3
 - Radarr version: 4.2.4.6635
 - Sonarr version: 3.0.9.1549
 - OS: Ubuntu 20 am7

**Additional context**
error logs:

```
Unexpected error in provider 'legendasdivx': Traceback (most recent call last):
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 173, in login
    bsoup = ParserBeautifulSoup(res.content, ['lxml'])
  File "/home/user/bazarr/bazarr/../libs/subliminal/providers/__init__.py", line 52, in __init__
    raise FeatureNotFoundbs4.FeatureNotFoundDuring handling of the above exception, another exception occurred:Traceback (most recent call last):
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/core.py", line 311, in list_subtitles_provider
    results = self[provider].list_subtitles(video, provider_languages)
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/core.py", line 247, in __getitem__
    provider.initialize()
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 160, in initialize
    self.login()
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 209, in login
    raise ServiceUnavailable("LegendasDivx.pt :: Uncaught error: %r", e)subliminal.exceptions.ServiceUnavailable: ('LegendasDivx.pt :: Uncaught error: %r', FeatureNotFound())|Traceback (most recent call last):
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 173, in login
    bsoup = ParserBeautifulSoup(res.content, ['lxml'])
  File "/home/user/bazarr/bazarr/../libs/subliminal/providers/__init__.py", line 52, in __init__
    raise FeatureNotFoundbs4.FeatureNotFoundDuring handling of the above exception, another exception occurred:Traceback (most recent call last):
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/core.py", line 311, in list_subtitles_provider
    results = self[provider].list_subtitles(video, provider_languages)
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/core.py", line 247, in __getitem__
    provider.initialize()
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 160, in initialize
    self.login()
  File "/home/user/bazarr/bazarr/../libs/subliminal_patch/providers/legendasdivx.py", line 209, in login
    raise ServiceUnavailable("LegendasDivx.pt :: Uncaught error: %r", e)
subliminal.exceptions.ServiceUnavailable: ('LegendasDivx.pt :: Uncaught error: %r', FeatureNotFound())
```

**Comment by morpheus65535** [comment]
You haven't intalled lxml properly. Make sure to deploy requirements.txt as instructed in wiki.

I've added a better description for next user who face this issue.

**Comment by jaum20** [comment]
installed, same error: Throttling legendasdivx for 20 minutes, until 22/12/13 13:41, because of: ServiceUnavailable. Exception info: 'LegendasDivx.pt :: Uncaught error: %r'

[bazarr.log](https://github.com/morpheus65535/bazarr/files/10220060/bazarr.log)

**Comment by morpheus65535** [comment]
I don't see the `FeatureNotFound` exception anymore in your log. Can you disable other providers, reset throttled providers, clear the log and search for a single subtitles?

BTW, the fix I made won't be included in Bazarr until v1.1.4-beta.3 is released.

**Comment by jaum20** [comment]
It was the lxml. Even installed, bs4 could not use it. I had to uninstall bs4 and install again to make it to work. Thank you
